rdrf/rdrf/forms/navigation/quick_links.py

```diff
@@ -385,14 +343,14 @@ def _build_menu(self):
             }
 
             # menu with everything, used for the admin page
-            if not settings.DESIGN_MODE:
-                MenuConfig().all = normal_menus
+            if not is_design_mode():
```

Comment:
`if/else` with a `not` condition. Any reason not to switch around and get rid of the `not`?

Or perhaps even better with less branching:

```python
self.menu.config.all = normal_menus
if is_design_mode():
    self.menu.config.all.update(design_menus)
```

Reply:
Updated.